reo/src/data_manager.py

```diff
@@ -1202,13 +1175,13 @@ def finalize(self):
         elec_chiller_cop = self.elecchl.chiller_cop if self.elecchl != None else 1.0
         absorp_chiller_cop = self.absorpchl.chiller_cop if self.absorpchl != None else 1.0
 
-        #TODO: handle chp parameter population directly in urdb_parse, or move all to data_manager
-        chp_thermal_prod_slope = tariff_args.chp_thermal_prod_slope if len(chp_techs) > 0 else []
-        chp_thermal_prod_slope_bau = tariff_args.chp_thermal_prod_slope_bau if len(chp_techs_bau) > 0 else []
-        chp_thermal_prod_intercept = tariff_args.chp_thermal_prod_intercept if len(chp_techs) > 0 else []
-        chp_thermal_prod_intercept_bau = tariff_args.chp_thermal_prod_intercept_bau if len(chp_techs_bau) > 0 else []
-        chp_fuel_burn_intercept = tariff_args.chp_fuel_burn_intercept if len(chp_techs) > 0 else []
-        chp_fuel_burn_intercept_bau = tariff_args.chp_fuel_burn_intercept_bau if len(chp_techs_bau) > 0 else []
+        # Specific attributes for CHP tech
+        chp_thermal_prod_slope = fuel_params.chp_thermal_prod_slope if len(chp_techs) > 0 else []
```

Comment:
could this logic be handled in FuelParams? We could also pass fuel burning techs into FuelParams, `fb_techs` on the develop branch, instead of all techs

Reply:
Yes, I made this update in [this commit](https://github.com/NREL/REopt_Lite_API/pull/124/commits/442d831a012daebf8280d1e47df4fd7479544cf3)